docs/spec/overview.md

```diff
@@ -9,9 +9,7 @@ backing implementations (much like the Kubernetes Ingress resource).
 - A **Subscription** describes the transformation of an event and optional
   forwarding of a returned event.
 
-- A **Source** allows an incoming events from an external system to be
-  _Subscribable_. A _Subscription_ is used to connect these events to
-  subsequent processing steps.
+- A **Source** emits incoming events to a _Channel_.
```

Comment:
:+1:

Reply:
This is fine as in source always puts things into a channel. However, I think it would be nice if a user wouldn't have to go digging into objects and then cut&paste these Channels around when creating pipelines. Hence the ability to deref, just like with Targetable, we could require a user to go in and at the lowest level just say it's a DNS name, same kind of applies here as in it's not ideal, if we think Targetable is a good concept, this is the same but in reverse.